app/models/miq_server/role_management.rb

```diff
@@ -123,14 +126,14 @@ def remove_master_for_role(server_role)
   end
 
   def check_server_roles
-    assigned_server_roles.each { |asr| asr.deactivate if asr.server_role.role_scope == 'zone' } if self.zone_id_changed?
+    assigned_server_roles.each { |asr| asr.deactivate if asr.server_role.role_scope == 'zone' } if zone_id_changed?
   end
 
   def server_role_names
     server_roles.pluck(:name).sort
   end
-  alias_method :my_roles,            :server_role_names
-  alias_method :assigned_role_names, :server_role_names
+  alias my_roles server_role_names
+  alias assigned_role_names server_role_names
```

Comment:
Sorry, I didn't see this sooner.

I don't want to bikeshed on this but I don't think `alias` vs. `alias_method` is worth discussing as part of these other very worthy changes.

https://github.com/rubocop-hq/ruby-style-guide/issues/821

Reply:
Note, we can discuss this in a followup. I don't want it to prevent the rest of the changes to go through.

Reply:
After doing some research, I'm not sure why rubocop has any preference at all. From what I've read `alias_method` is actually more flexible, though near as I can tell it's basically irrelevant for the way we're using it.

Reply:
Exactly.  It's good to keep rubocop honest.  I think `alias_method` or no preference would be my choices.

Reply:
Only thing I can think of is that maybe `alias_method` is less memory efficient since it means adding more symbols, though I've never looked at the implementation for either, so I couldn't say one way or another.

Reply:
Here was their reasoning: https://github.com/rubocop-hq/ruby-style-guide#alias-method-lexically

Reply:
Ok, undone.